Tests/eppo/ClientTests.swift

```diff
@@ -20,13 +20,30 @@ class EppoMockHttpClient: EppoHttpClient {
     public func post() throws {}
 }
 
-class AssignmentLoggerSpy {
+class EppoJSONAcceptorClient: EppoHttpClient {
+    var jsonResponse: Data?
+    
+    init(jsonResponse: String) {
+        self.jsonResponse = jsonResponse.data(using: .utf8)
+    }
+    
+    func get(_ url: URL) async throws -> (Data, URLResponse) {
+        guard let jsonData = jsonResponse else {
+            throw URLError(.badServerResponse)
+        }
+        return (jsonData, URLResponse(url: url, mimeType: "application/json", expectedContentLength: jsonData.count, textEncodingName: nil))
+    }
+}
```

Comment:
directly provide a JSON string instead of loading from disk like the other mock - this is useful for changing the configuration within a test, as I do below.